Cargo.toml

```diff
@@ -29,10 +29,13 @@ k8s-openapi = { version = "0.19", default-features = false, features = ["v1_27"]
 kube = { version = "0.85", default-features = false, features = ["client", "rustls-tls"] }
 lazy_static = "1.4"
 nu-ansi-term = "0.49"
+once_cell = "1.18"
 phf = "0.11"
 phf_codegen = "0.11"
 rand = "0.8"
-reqwest = { version = "0.11.16", default-features = false, features = ["rustls-tls"] }
+regex = "1.9"
+# We use the feature "rustls-tls-native-roots" instead of "rustls-tls" to avoid pulling in "webpki-roots" and hence MPL-2.0: https://github.com/rustls/webpki-roots/blob/main/LICENSE
```

Comment:
Any particular reason to switch back to `rustls-tls-native-roots`? Or is this due to git merge stuff going on?

Reply:
Oh, my bad. I applied the wrong changes when merging. Will update asap.

Reply:
Reverted in 8e2100f

Reply:
Awesome, thanks!